components/nexus-security/src/main/java/org/apache/shiro/nexus5727/FixedDefaultWebSessionManager.java

```diff
@@ -36,15 +36,20 @@
 public class FixedDefaultWebSessionManager
     extends DefaultWebSessionManager
 {
+  /**
+   * Default HTTP session cookie name that Nexus uses - NXSESSIONID
+   */
+  public static final String DEFAULT_NEXUS_SESSION_COOKIE_NAME = "NXSESSIONID";
+
   private static final Logger log = LoggerFactory.getLogger(FixedDefaultWebSessionManager.class);
 
   @Inject
   public void configureProperties(
       final @Named("${shiro.globalSessionTimeout:-" + DEFAULT_GLOBAL_SESSION_TIMEOUT + "}") long globalSessionTimeout,
-      final @Named("${shiro.secureSessionCookies:-false}") boolean secureSessionCookies)
```

Comment:
I think we should leave this setting in place, as it provides a way to enforce the secure flag across all connections which isn't covered by the dynamic filter.